src/mscorlib/src/System/String.cs

```diff
@@ -293,6 +293,23 @@ unsafe internal int GetBytesFromEncoding(byte* pbNativeBuffer, int cbNativeBuffe
             }            
         }
 
+        unsafe internal int GetByteCountFromEncoding(int index, int count, Encoding encoding)
+        {
+            fixed (char* pwzChar = &this.m_firstChar)
```

Comment:
This should be in the Encoding.cs. There is no reason for this to be in string.cs